OutOfSchool/OutOfSchool.DataAccess/Models/Direction.cs

```diff
@@ -17,5 +17,7 @@ public class Direction : IKeyedEntity<long>
     [MaxLength(500)]
     public string Description { get; set; } = string.Empty;
 
+    public bool IsDeleted { get; set; }
```

Comment:
(Design considerations) As I can see it's common pattern to have `IsDeleted` property as a flag. Also there are lot of code checking it for this and other similar DTOs. Isn't it a reason to separate and use `ISoftDeleted` (or suggest your name) interface?

Reply:
Ok. We can use it in the EntityRepositoryBase to add predicate for queries. If you have another view how we can use this interface, please, write it. Thanks.